packages/appstore/src/components/modals/real-wallets-upgrade/real-wallets-upgrade.tsx

```diff
@@ -30,6 +31,13 @@ const RealWalletsUpgrade = observer(() => {
 
     const handleClose = () => toggleWalletsUpgrade(false);
 
+    const { start } = useWalletMigration();
+
+    const upgradetoWallets = () => {
+        start();
+        toggleWalletsUpgrade(false);
```

Comment:
@thisyahlen-deriv Do we still need things like `is_real_wallets_upgrade_on`, `toggleWalletsUpgrade`, `is_wallet_migration_failed`, `setWalletsMigrationFailedPopup`?

Reply:
This is_wallet_migration_failed, setWalletsMigrationFailedPopup ones will be covered in another user story i guess

For the rest, we need it toggle the modal 👀